aten/src/ATen/native/SharedReduceOps.h

```diff
@@ -157,10 +157,10 @@ struct MeanOps {
   }
 };
 
-template <typename acc_t>
+template <typename scalar_t, typename acc_t=scalar_t>
```

Comment:
Add a comment here explaining the template and its intended use.

The use of `acc_t` seems accurate here but is different from the typical notion of an accumulate type:

https://github.com/pytorch/pytorch/blob/324c18fcad579b1afa63ae45528bf598ba8ec4ca/aten/src/ATen/AccumulateType.h#L21

This PR should add comments as needed to make this distinction clear to avoid confusion.